index.html

```diff
@@ -10,9 +10,10 @@
         <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo">
         <nav id="nav-bar">
             <a class="nav-link" href="#features">Features</a>
-            <a class="nav-link" href="#howitworks">How it works</a>
+            <a class="nav-link" href="#howitworks">How does it works</a>
             <a class="nav-link" href="#pricing">Pricing</a>  
         </nav>
+
```

Comment:
Please use Semantic tags for html

Reply:
- [ ] Kindly consider giving a reason for this change, it will help the student understand why the change is suppose to be done and how it should be done. 
- [ ] Moreover, you have to make it clear and understandable by giving details. For instance you ask the student to use semantic tags for html, but the student use nav and a tags which are also semantics tags. So what tags? Please be clear and specific. :pray:

Reply:
- [ ] Same thing as said by the previous reviewer, please be more detailed, the student needs to know why he is asked to implement any of the changes, please